### PR TITLE
i18n(zh-cn): update `api-reference.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

update `api-reference.mdx`

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9081
- https://github.com/withastro/docs/pull/9128

### CODE REVIEW

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 1]
```diff
@@ -503,7 +503,9 @@ return Astro.rewrite(new Request(new URL("../", Astro.url), {
 
 [URL](https://developer.mozilla.org/zh-CN/docs/Web/API/URL )对象，由当前 `Astro.request.url` 的链接字符串值构成。对于与请求的链接的个别属性进行交互是有用的，如路径名和起源。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/api-reference.mdx:504]:
```suggestion
该 [URL](https://developer.mozilla.org/zh-CN/docs/Web/API/URL ) 对象，是由当前 `Astro.request.url` 的链接字符串值构造。对于与请求 URL 的各个属性进行交互时非常有用，例如路径名和来源。
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]
有一些之前的旧翻译不太通顺，可以也在这其中更新吧？

File: src/content/docs/zh-cn/reference/api-reference.mdx (hunk 1 above)
Comment (liruifengv) [reply src/content/docs/zh-cn/reference/api-reference.mdx]: 我改了一下

**Review by Nin3lee — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job! 🙌

**Pull request merged**